src/resources.cpp

```diff
@@ -118,15 +122,52 @@ P<ResourceStream> DirectoryResourceProvider::getResourceStream(string filename)
 std::vector<string> DirectoryResourceProvider::findResources(string searchPattern)
 {
     std::vector<string> found_files;
+#if defined(ANDROID)
+    //Limitation : 
+    //As far as I know, Android NDK won't provide a way to list subdirectories
+    //So we will only list files in the first level directory 
+    ANativeActivity *nactivity {sf::getNativeActivity()};
+   
+    AAssetManager *asset_manager{nullptr};
+    if(nactivity)
+    {
+        asset_manager = nactivity->assetManager;
+    }
+    if(asset_manager)
+    {
+        int idx = searchPattern.rfind("/");
+        string forced_path = basepath;
+        string prefix = "";
+        if (idx > -1)
+        {
+            prefix = searchPattern.substr(0, idx);
+            forced_path += "/" + prefix;
+            prefix += "/";
+        }
+        AAssetDir* dir = AAssetManager_openDir(asset_manager, (forced_path).c_str());
+        if (dir)
+        {
+            AAssetDir_rewind(dir);
```

Comment:
I forgot to remove this after pulling hair on why I would not get my directory... Removing it, it's useless.